client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx

```diff
@@ -244,11 +244,26 @@ const MarketplacePluginInstall = ( { productSlug }: MarketplacePluginInstallProp
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [ pluginActive, automatedTransferStatus ] ); // We need to trigger this hook also when `automatedTransferStatus` changes cause the plugin install is done on the background in that case.
 
-	const steps = [
-		isUploadFlow ? translate( 'Uploading plugin' ) : translate( 'Setting up plugin installation' ),
-		translate( 'Installing plugin' ),
-		translate( 'Activating plugin' ),
-	];
+	const steps = useMemo(
```

Comment:
Is `useMemo` required here? The linter is complaining about `translate` missing in the dependencies array.

Reply:
Yes, it is needed because this value is being used as a dependency on the other component and cannot be recreated every time. I'll add `translate` as a dependency.

Reply:
Updated at https://github.com/Automattic/wp-calypso/pull/64087/commits/785ebedbe944dc66f8248a8f2902be5b5508e207